a.tsv

```diff
@@ -307,7 +307,7 @@ A13	English Opening: Agincourt Defense, Wimpy System	1. Nf3 Nf6 2. c4 e6 3. b3 d
 A13	English Opening: Neo-Catalan	1. Nf3 Nf6 2. c4 e6 3. g3 d5
 A13	English Opening: Neo-Catalan Declined	1. Nf3 Nf6 2. c4 e6 3. g3 d5 4. Bg2 Be7
 A13	English Opening: Romanishin Gambit	1. Nf3 Nf6 2. c4 e6 3. g3 a6 4. Bg2 b5
-A13	Horwitz Defense: Zilbermints Gambit	1. c4 e6 2. d4 e5
+A13	Horwitz Defense: Zilbermints Gambit	1. d4 e6 2. e4 e5
```

Comment:
hi i think d4 and c4 has to be there according to chesstempo and horwitz is under A40